test/functional/ios/keyboard_tests.py

```diff
@@ -52,7 +52,7 @@ def test_hide_keyboard_no_key_name(self):
 
         el = self.driver.find_elements_by_class_name('XCUIElementTypeTextField')[0]
         el.click()
-        el.set_value('Testing')
+        el.send_keys('Testing')
```

Comment:
Use `send_keys` randomly as part of functional test

Reply:
it makes sense to provide a path to an existing temporary file to make sure we handle these properly